packages/main/src/compose/newConversation.test.ts

```diff
@@ -0,0 +1,45 @@
+import * as cache from "../cache"
+import { inbox } from "../cache/testFixtures"
+import db from "../db"
+import { MessageInput } from "../generated/graphql"
+import { composeNewConversation } from "./newConversation"
+
+let account: cache.Account
+let accountId: cache.ID
+let boxId: cache.ID
+
+beforeEach(async () => {
+  const { lastInsertRowid } = db
+    .prepare("insert into accounts (email) values (?)")
+    .run("jesse@sitr.us")
+  accountId = lastInsertRowid
+  account = cache.getAccount(accountId)!
+  boxId = cache.persistBoxState(accountId, inbox)
+})
+describe("a message for a new Conversation", () => {
+  it("has a content-ID", async () => {
```

Comment:
Good stuff! But as with the other tests we do not need to specify `partID` or `params` in the expected object.